### PR TITLE
Adds initial setup for functions app

### PR DESCRIPTION
**Pull request by DannyGB**
There is a current limitation where the RunHealthCheckCommand class is replicated into this repo and in the API repo. When we have a messages nuget package this will be refactored out to use that

### CODE REVIEW

**Review by chrisfoster76 — APPROVED** [review]

File: pipeline-templates/job/code-build.yml [hunk 1]
```diff
@@ -1,47 +1,47 @@
...
   variables:
   - group: BUILD Management Resources
+  workspace:
+    clean: all
   steps:
-    - task: gitversion/setup@0
-      displayName: gitversion/setup
-      inputs:
-        versionSpec: 5.x
+  - task: UseGitVersion@5
```
Comment (nbowes24) [inline pipeline-templates/job/code-build.yml:13]: I believe there is a step template for it now

File: pipeline-templates/job/code-build.yml [hunk 2]
```diff
@@ -1,47 +1,47 @@
 parameters:
   SolutionBaseName:
-  BuildConfiguration:
 
 jobs:
 - job: CodeBuild
+  pool:
+    name: DAS - Continuous Integration
```
Comment (nbowes24) [inline pipeline-templates/job/code-build.yml:7]: This should use the new ubuntu 20 agents

Comment (cpwevans) [reply pipeline-templates/job/code-build.yml]: I've now changed the pool being used and added the demand to specify Ubuntu 20

File: pipeline-templates/job/code-build.yml (hunk 1 above)
Comment (cpwevans) [reply pipeline-templates/job/code-build.yml]: I've updated this with the template step from the building block.

**Review by nbowes24 — APPROVED** [review]

**Pull request merged**